cul.js

```diff
@@ -281,24 +323,27 @@ function connect() {
 
 }
 
-function insertObjects(objs, cb) {
-    if (objs && objs.length) {
-        var newObject = objs.pop();
+//function insertObjects(objs, cb) {
+//    if (objs && objs.length) {
+//        var newObject = objs.pop();
 
-    } else if (cb) {
-        cb();
-    }
-}
+//    } else if (cb) {
+//        cb();
+//    }
+//}
 
 function main() {
-
-    adapter.getForeignObject('cul.meta.roles', function (err, res) {
+    
+    // in this template all states changes inside the adapters namespace are subscribed
+    adapter.subscribeStates("*");
+    
+    adapter.objects.getObject('cul.meta.roles', function (err, res) {
```

Comment:
do not user adapter.objects! This is deprecated! please use the method directly from adapter class and for "getObjectview" also and add a js-controller >=2.1.1 dependency please

Reply:
This line should be reverted to getForeignObject

Reply:
done